tests/resources/test_profiler.py

```diff
@@ -14,6 +16,7 @@ def setUp(self):
         super().setUp()
         self.web = StubSite(ProfilerResource(self.manager))
 
+    @pytest.mark.skipif(sys.platform == 'win32', reason='shutil.rmtree fails on Windows')
```

Comment:
Maybe this link is useful to get it fixed? https://docs.python.org/3.6/library/shutil.html#rmtree-example

Reply:
I actually haven't debugged it at all. I don't know if it's worth the time, I think we can leave this to a separate task.

Reply:
The reason that I'm not too worried about getting this to work is that I've never gotten rocksdb to work on Windows (at the very least it will need an update on the python lib, which isn't being well maintained so we might have to fork it), and this might make Windows not viable for the size of the database we have today (unless one uses a lot of RAM and waits for a full sync). So I'm taking it slow and only putting the absolute minimal amount of effort in this.

Reply:
Ok. In this case, I think we should open an issue to remove these skips and fix theses tests in the future.

Reply:
Created #260 to track this.